clouds/databricks/common/python_utils/create_drop_functions.py

```diff
@@ -1,13 +1,22 @@
 import os
+import sys
 
 
-python_util_path = os.path.dirname(os.path.realpath(__file__))
+if len(sys.argv) < 3 or len(sys.argv) > 4:
+    raise Exception('Parameters: sql-directory, output-file[, schema]')
+
+
+sql_path = sys.argv[1]
+output_file = sys.argv[2]
+schema = sys.argv[3] if len(sys.argv) > 3 else None
 
 
 def get_uninstall_for_module(module_path, queries_list):
-    database = os.getenv('DB_SCHEMA')
+    database = schema
     database = 'default' if database is None else database
     _, _, functions = next(os.walk(module_path))
+    # FIXME: should we read the SQL file, extract the @@DB_SCHEMA@@.function_name
+    # frrom the definition and substitute the schema?
```

Comment:
Actually if you take a look to `DROP_FUNCTIONS.sql` in other providers it drops all the functions installed by schema. It doesn't care about the sql files that we have in the repo and doesn't read them (BTW there is one example of extracting UDFS and procedures in core/clouds/snowflake/common/build_share.js). This was done to avoid keeping private functions that has no named files and public functions that are renamed.

Reply:
Yep, you're right, we'll do it in a separte ticket